### ISSUE TITLE
feature request: Finer control of "stop on track" commands

### ISSUE THREAD

**Issue by ghost** [clementine-player/Clementine#4683]
Currently the non-modifiable shortcut `Ctrl+Alt+V` only triggers 'stop on currently playing track'. However, it's not possible (at least in Ubuntu) to set a key for 'stop on currently selected track'.

E.g. the following workflow:
1. Queue
2. Queue
3. Queue
4. Ctrl+Alt+V
5. :crying_cat_face: hit Ctrl+Alt+V again
6. Move hand from keyboard to clicker
7. Right click the currently selected track, click "stop on this track"

There are a couple features I can think of that could alleviate this:
- Make "stop on selected track" customizable
- Change "stop on this track" to "stop on selected track"

**Comment by ArnaudBienner** [comment]
I don't think this can be done this way (what if the selected track will never be played because it appears before the currently playing song?)
But maybe having a "stop when queue will be empty" would make sense, though a simple workaround is to just create a playlist with only the songs you want to listen to (and "repeat playlist" turned off, of course)

**Comment by talexand** [comment]
'stop on currently playing track' can produce the same behavior you describe, actually: just double-click another track that comes later in the playlist, and the stop tag loses its effect. The GUI isn't confusing about it though, so I don't think it's a surprise for it to be this way.

A 'stop on end of queue' would of course also be wonderful.
